ucm2/USB-Audio/USB-Audio.conf

```diff
@@ -28,6 +28,17 @@ If.realtek-alc1220-vb {
 	True.Define.ProfileName "Realtek/ALC1220-VB-Desktop"
 }
 
+If.realtek-alc4080 {
+
+	Condition {
+		Type RegexMatch
+		String "${CardComponents}"
+		# 0db0:419c MSI MPG X570S Carbon Max Wifi
+		Regex "USB(0db0:419c)"
```

Comment:
Please make it `USB((0db0:419c)|(0b05:1996))` to match ASUS ROG Maximus XIII

Reply:
ALC4080 with the ID `0b05:1996` is used by other ASUS motherboards as well, so it's not specific to just Maximus XIII.

Reply:
On the MSI Edge Wifi Z690 motherboard the ALC4080 is: `0db0:1feb`

Reply:
@mavoga @tekstryder btw. for proper Control Names I supplied a patch for my board to the Linux kernel: https://github.com/torvalds/linux/commit/5762f980ca10dcfe5eead7c40d1c34cae61f409b

Does Speaker/Front Headphone Volume control work for you? Maybe you will need to use the same map for the IDs of your boards.

Reply:
@lordhoto I patched `sound/usb/mixer_maps.c` to apply the same map to the USB id of my card too, therefore the volume controls work perfectly (currently trying to figure out how to correctly send a kernel patch since I never did it before...).

Reply:
Hi. For msi mag Z690 TomaHawk wifi ddr4 `0db0:b202`

Reply:
Hi, for MAG X570S TOMAHAWK MAX WIFI is  `0db0:151f`

Reply:
> btw. for proper Control Names I supplied a patch for my board to the Linux kernel:

@lordhoto I am trying to patch the profile for MSI MPG Z690 FORCE WIFI, but simply adding USB Audio ID from `lsusb` output does not seem to enable front headphone and mic. How the kernel patch is related to this patch? Do I understand correctly that without applying the similar kernel patch for each motherboard listed in "If.realtek-alc4080" the config is not expected to work?